themes/stanford_basic/stanford_basic.theme

```diff
@@ -468,3 +468,23 @@ function stanford_basic_theme_suggestions_views_view_list_alter(array &$suggesti
   $suggestions[] = "views_view_list__$view_id";
   $suggestions[] = "views_view_list__{$view_id}__$display_id";
 }
+
+/**
+ * Implements hook_preprocess_paragraph().
+ */
+function stanford_basic_preprocess_paragraph(&$vars) {
+  $paragraph = $vars['elements']['#paragraph'];
+  if ($paragraph->getType() == 'stanford_spacer') {
+   $vars['attributes']['class'][] = 'paragraph--type--stanford-spacer';
+    if (!$paragraph->isPublished()) {
+      $vars['attributes']['class'][] = 'paragraph--unpublished';
+    }
+    if ($paragraph->hasField('su_spacer_size') &&
+      !$paragraph->get('su_spacer_size')->isEmpty() &&
+      $paragraph->get('su_spacer_size')->getString() != '_none' ) {
+      $vars['attributes']['class'][] = $paragraph->get('su_spacer_size')->getString();
+    } else {
+      $vars['attributes']['class'][] = 'su-spacer-default';
+    }
+  }
```

Comment:
All of this is overly complicated. 
- `paragraph--type--stanford-spacer` is already available in the markup.
- you don't need `paragraph--unpublished`
- you don't need to check for `_none`
- You don't need `su-spacer-default`

This hook should be about 4 lines long to accomplish the desired result.